client/client.go

```diff
@@ -579,6 +587,17 @@ type Security struct {
 	SSLEngine      string `json:",omitempty"`
 }
 
+// Connectivity instructs the client how we want to treat the keep alive connections
+// and when a connection is considered idle and therefore closed
+type Connectivity struct {
+	// if set to true, there will be no persistent connections, and every
+	// HTTP transaction will try to establish a new connection
+	DisableKeepAlive bool `json:",omitempty"`
+	// a number of seconds after which a connection is considered idle and closed
+	// the longer this is the longer connections are up after the first call over HTTP
```

Comment:
Nitpick: It took me a while to understand what this meant because it looks like one sentence stretched over two lines, but it's really two sentences. Please add a period after "closed", and even capital letters would be nice! :-)

Reply:
done.